javaslang/src/main/java/javaslang/collection/Tree.java

```diff
@@ -11,12 +11,32 @@
 import javaslang.collection.List.Nil;
 import javaslang.collection.Tree.Empty;
 import javaslang.collection.Tree.Node;
-import javaslang.collection.TreeModule.*;
+import javaslang.collection.TreeModule.FlatMap;
```

Comment:
Currently most of us use IntelliJ IDEA's "organize imports" feature. The IDAE settings are included in the repo (see folder `.ide`). I have no strong opinion regarding imports. The generated sources' import section differs too.